src/infrastructure/managing-items.md

```diff
@@ -73,38 +73,55 @@ The necessary context object is available in all controllers and clients via `$t
 
 ## Create item
 
-Creating a new empty item is the easiest part. Simply call *create() of the manager you need a new item from:
+Creating a new empty item is the easiest part. Simply call the `create()` method of the manager you need a new item from:
 
 ```php
 $item = $manager->create();
 $item = $manager->create( [/*...*/] );
 ```
 
-This creates a new object with some default values for the item set. You can add properties if you pass a list of key/value pairs in the same structure as returned by toArray() of the item as first argument. The item doesn't have an unique ID by default and is only available in memory.
+This creates a new object with some default values for the item set. You can add properties if you pass a list of key/value pairs:
+
+```php
+$item => $manager->create( [$key1 => $value1, $key2 => $value2] )
+
+// Example for a product item:
+$item => $manager->create( ['product.code' => 'test', 'product.label' => 'Test'] )
+```
+
+If you want to find out more about the structure of an item's arguments, use `toArray()`:
```

Comment:
"Structure" is misleading too, better to speak of "keys"

Reply:
I suggest to leave this part of the docu as it was or you provide a new text snippet in case you think it is really relevant. I am clearly not familiar enough with that area of aimeos to provide correct documentation.

Reply:
Thanks for the feedback! 
New commit added with your suggested changes.